style/index.css

```diff
@@ -50,3 +52,381 @@
   padding: 12px 0 4px 12px;
   text-transform: uppercase;
 }
+
+.jp-TableOfContents-codeContainer {
+  overflow: hidden;
+}
+
+.jp-TableOfContents-code {
+  font-size: 9px;
+  max-height: 70px;
+}
+
+.cm-toc .CodeMirror {
+  font-size: 9px;
+  z-index: 0;
+  border: var(--jp-border-width) solid var(--jp-cell-editor-border-color);
+  border-radius: 0px;
+  background: var(--jp-cell-editor-background);
+  max-width: 100%;
+  max-height: 36px;
+}
+
+.toc-code-span {
+  width: 100%;
+  max-width: 100%;
+  overflow: hidden;
+}
+
+.cm-toc .CodeMirror-scroll {
+  overflow-x: hidden;
+  overflow-y: hidden;
+}
+
+.markdown-cell {
+  overflow-x: hidden;
+  overflow-y: hidden;
+  max-height: 22px;
+}
+
+.markdown-cell code {
```

Comment:
Many of these CSS classes have pretty general names. I worry that some could stomp on other CSS on the page. Can we prefix them with some common prefix, like `jp-toc` so that we know we are being good CSS citizens?